core-web/libs/edit-content/src/lib/fields/dot-edit-content-relationship-field/dot-edit-content-relationship-field.component.ts

```diff
@@ -98,23 +120,41 @@ export class DotEditContentRelationshipFieldComponent implements ControlValueAcc
     $field = input.required<DotCMSContentTypeField>({ alias: 'field' });
 
     /**
-     * A computed signal that holds the content type id for the relationship field.
-     * This id is used to get the content type fields.
+     * Creates an instance of DotEditContentRelationshipFieldComponent.
+     * It sets the cardinality of the relationship field based on the field's cardinality.
+     *
+     * @memberof DotEditContentRelationshipFieldComponent
      */
-    $contentTypeId = computed(() => {
-        const field = this.$field();
+    constructor() {
+        effect(
+            () => {
+                const field = this.$field();
 
-        return field?.relationships?.velocityVar || null;
-    });
+                const cardinality = field?.relationships?.cardinality ?? null;
+
+                if (cardinality === null) {
+                    return;
+                }
+
+                this.store.setCardinality(cardinality);
+            },
+            {
+                allowSignalWrites: true
+            }
```

Comment:
I thought this was enabled by default

Reply:
Is correct in v19 this option is not necessary, but now with v18 we need this flag.